src/app/components/core/configuration-manager/list-type-configuration/list-type-configuration.component.html

```diff
@@ -0,0 +1,39 @@
+<div [formGroup]="listItemsForm">
+  <ng-container *ngFor="let item of listItems.controls; let i = index">
+    <div formArrayName="listItems" class="field">
+      <div class="columns">
+        <div class="column is-11">
+          <input *ngIf="configuration.items == 'integer'" class="input is-small" type="text" formControlName="{{i}}"
+            [minValue]="configuration?.minimum" [maxValue]="configuration?.maximum" appIntegerOnly>
+          <input *ngIf="configuration.items == 'float'" class="input is-small" type="text" formControlName="{{i}}"
+            [minValue]="configuration?.minimum" [maxValue]="configuration?.maximum" appFloatOnly>
+          <input *ngIf="configuration.items == 'string'" class="input is-small" type="text" formControlName="{{i}}"
+            [attr.maxLength]="configuration?.length">
+        </div>
+        <div class="column pl-0 pt-2">
+          <span class="icon" *ngIf="i != 0">
+            <button class="delete is-small" (click)="removeListItem(i)"></button>
+          </span>
```

Comment:
Are we using bi-x-circle or bi-trash icon?

Reply:
You can ignore if this is the same, We have in Control Pipelines KV for Parameters.

Reply:
This is same for CP KV for Parameters.